src/sema/dotgraphviz.rs

```diff
@@ -2072,8 +2072,8 @@ impl Namespace {
             for decl in &self.enums {
                 let mut labels = vec![String::new(); decl.values.len()];
 
-                for (name, (_, pos)) in &decl.values {
-                    labels[*pos] = format!("value: {}", name);
+                for (pos, value) in decl.values.iter().enumerate() {
+                    labels[pos] = format!("value: {}", value.0);
```

Comment:
It looks like this could be written without a for loop with `.collect::<Vec<String>>()`.

Reply:
Let me check.

Reply:
Fixed